kafka-ui-api/Dockerfile

```diff
@@ -1,7 +1,6 @@
-FROM alpine:3.15.0
+FROM openjdk:13.0.2-jdk-buster
```

Comment:
Why is this necessary?

Can we create a new Dockerfile so that we don't have conflicts with upstream master if they change this file?

Reply:
We need this because the Java grpc package relies on stuff that alpine is missing. All make a new clst.Dockerfile though